.github/workflows/docs/BuildAndPublish.md

```diff
@@ -0,0 +1,173 @@
+
+#### MacOS Notarization
+
+By default, your built game on MacOS will be flagged as dangerous and players will need to allow its execution by going into.
+
+To avoid this, you need to notarize it, i.e. tell Apple who you are and what is your binary.
+
+For that, you'll need first to create an Apple developer account (99USD/year). Then, you'll need to adapt the Export configuration of MacOS [using this guide](https://docs.godotengine.org/en/latest/tutorials/export/exporting_for_macos.html#if-you-have-an-apple-developer-id-certificate-and-exporting-from-linux-or-windows) to add **rcodesign** notarization and your Apple tokens.
+
+## Setup Itch.io publication
+
+### 1. Create a new project on itch.io
+
+Go to [itch.io], click on the top right and **Upload a New Project**. Fill in the game name and any information you want, but don't upload any file. **Save the project as a Draft**.
+
+### 2. Create a `ITCH_USERNAME` and `ITCH_GAME` variable
+
+To find your itch.io username and the name of your game, look at the url of your project: `https://your-username.itch.io/your-game`. The username is the first part of the URL, and the game name is in the
+
+Then, go to your Github repository Settings > Secrets and Variables > Actions. Then, select the **Variable** tab.
+
+Create two variables: `ITCH_USERNAME` and `ITCH_GAME`. You should have something like this (with your real username and your real game name instead):
+
+![github variables](./img/github-variables.png)
+
+### 3. Create a `BUTLER_API_KEY` Github secret
+
+1. Install [butler.](https://itch.io/docs/butler/installing.html) This is the official CLI tool for itch.io
+
+2. Unzip and make sure the bin is executable
+
+```bash
+chmod +x butler
+```
+
+3. Run
+
+```bash
+butler login
+```
+
+This should open your browser. Login and allow butler to access your account.
+
+![Authorize butler](./img/authorize_butler.png)
+
+In the terminal, the login flow will conclude with something like this:
+
+```
+Authenticated successfully! Saving key in /Users/username/Library/Application Support/itch/butler_creds...
+```
+
+4. Get your butler API key by reading the content of this file. Beware of spaces in the filepath! This will show you a 40 characters string which is your butler API key.
+
+```bash
+cat "/Users/username/Library/Application Support/itch/butler_creds"
+```
+
+> **Warning:** your butler API key is sensitive and secret. Do not share it with anyone and **do not** commit it to your repository and **do not** add it directly to the workflow file.
+
+5. [Create a new Github secret](https://docs.github.com/en/actions/how-tos/write-workflows/choose-what-workflows-do/use-secrets) for your Github repository by going into Settings > Secrets and Variables > Actions and selecting the Secrets tab. Call the secret `BUTLER_API_KEY` and inside, paste the result of the previous step.
+
+![github secrets](./img/github-secrets.png)
+
+### 4. Setup the HTML / Playable version of your game on itch
```

Comment:
This section jumps from pre-deploy setup to assuming that the user already deployed to the itch.io page. This is where I'd suggest going into creating a Release and triggering the Action. Maybe mention the ability to monitor actions and check for errors from the **Actions** tab.